db/migrate/20150615075515_create_course_events.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCourseEvents < ActiveRecord::Migration
+  def change
+    create_table :course_events do |t|
+      t.string :location
+      t.integer :event_type, default: 0
+
+      t.timestamps null: false
```

Comment:
It is already accounted for since Course::Event already acts_as_lesson_plan_item ?